src/shared/widgets/floating/static-doorhanger.jsx

```diff
@@ -16,13 +16,17 @@ import CSSModules from 'react-css-modules';
 import WidgetComponent from '../helpers/widget-component';
 import Styles from './static-doorhanger.css';
 import Tooltip from './tooltip';
-import CSSArrow from '../css-arrow';
```

Comment:
Eh, it might be useful in the future? I'm ok with removing it if you feel strongly about it.